homeworks/01-twitch/README.md

```diff
@@ -48,6 +49,7 @@ public interface StreamingPlatform {
      * @param username the username of the streamer
      * @param stream   the stream to end
      * @return the created {@link Video}
+     * @throws IllegalArgumentException if any of the parameters are null or empty
```

Comment:
Maybe change it to "if any of the parameters are null (or a String is empty)" to avoid ambiguity, what does empty mean for a Stream.